src/subpages/transaction.tsx

```diff
@@ -94,7 +94,10 @@ const TransactionPage: React.FC<TransactionPageProps> = ({ location }) => {
                         <dt>{argument.key}</dt>
                         <dd>
                           <pre>
-                            <code> {JSON.stringify(argument.value, null, 2)} </code>
+                            <code>
+                              {' '}
```

Comment:
Just curiosity, why we need this string(`" "`) here?

Reply:
This is formated by prettier. I'll fix it. 😅